docs/install.md

```diff
@@ -126,10 +128,16 @@ To install the prebuilt sharp binaries from a directory on the local filesystem,
 set the `sharp_local_prebuilds` npm config option
 or the `npm_config_sharp_local_prebuilds` environment variable.
 
+### Prebuilt libvips binaries
+
 To install the prebuilt libvips binaries from a custom URL,
 set the `sharp_libvips_binary_host` npm config option
 or the `npm_config_sharp_libvips_binary_host` environment variable.
 
+To install the prebuilt libvips binaries from a directory on the local filesystem,
+set the `sharp_local_libvips_dir` npm config option
```

Comment:
Would naming this something like `sharp_libvips_local_prebuilds` provide a more consistent approach?

Reply:
updated